Description: Control modifier appears as Primary when setting shortcuts with gtk >= 2.24.7
Author: Yves-Alexis Perez <corsac@debian.org>
Bug: https://bugzilla.xfce.org/show_bug.cgi?id=8056

Index: libxfce4ui-4.9.0/libxfce4kbd-private/xfce4-keyboard-shortcuts.xml
===================================================================
--- libxfce4ui-4.9.0.orig/libxfce4kbd-private/xfce4-keyboard-shortcuts.xml	2011-01-16 16:01:09.000000000 +0100
+++ libxfce4ui-4.9.0/libxfce4kbd-private/xfce4-keyboard-shortcuts.xml	2011-11-28 16:04:43.777907660 +0100
@@ -10,10 +10,10 @@
       <property name="&lt;Alt&gt;F3" type="string" value="xfce4-appfinder">
         <property name="startup-notify" type="bool" value="true"/>
       </property>
-      <property name="&lt;Control&gt;&lt;Alt&gt;Delete" type="string" value="xflock4"/>
+      <property name="&lt;Primary&gt;&lt;Alt&gt;Delete" type="string" value="xflock4"/>
       <property name="XF86Display" type="string" value="xfce4-display-settings --minimal"/>
       <property name="&lt;Super&gt;p" type="string" value="xfce4-display-settings --minimal"/>
-      <property name="&lt;Control&gt;Escape" type="string" value="xfdesktop --menu"/>
+      <property name="&lt;Primary&gt;Escape" type="string" value="xfdesktop --menu"/>
       <property name="XF86WWW" type="string" value="exo-open --launch WebBrowser"/>
       <property name="XF86Mail" type="string" value="exo-open --launch MailReader"/>
     </property>
